docs/application/dotnet/guides/nui/resources.md

```diff
@@ -3,14 +3,48 @@
 NUI provides several ways to handle resource images.
 This tutorial describes the use of Image resources in NUI.
 
-## Overview
-
-Resources in NUI can apply Images or three-dimensional Models.
-
-The common method to access these resources is through Visuals.
-Controls use these visuals to display what is required.
-
-See [ImageView tutorial](imageview.md).
+## Image handling methods
+
+The two basic methods of handling image resources in NUI are as follows:
+ 
+1. `Visuals` are the main building block for UI components and provide reusable rendering logic that is controlled using properties. For more information, see [Visuals tutorial](visuals.md) page:
```

Comment:
@mijin85cho I had checked out this [page](https://github.com/Samsung/tizen-docs/blob/b42141a11efa2b5c15fb394823baa2092436e442/docs/application/dotnet/guides/nui/visuals.md) for use of "s" .

Should we then not change the H1 as per your suggestion to maintain consistency.

Reply:
@nawab-samsung In the reference page, it is not using code tag, thus it is calling every items which belong to "Visual".
(Border, Color, Gradient, Image.... etc as in the following content.)

At here code tag is applied so it can be considered as https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Visual.html,
the specific Visual struct in the API.
In that case we should use the accurate name, which is used in the API reference.

Please consider that the meaning is slightly different, to prevent the unconditional applying consistency without checking the meaning.

Reply:
@mijin85cho 

1. Can you please help me understand how is https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Visual.html,  link related to present context ?

I would like to learn from you. 

Border, Color, Gradient, Image) are collectively called as visuals.

Here we are not talking of just one visual but we are talking of more than one. 

When we talk of visual as stated in the following text: 

 A visual must be defined for each state and substate. 

That means visual is some property or something. 

But if you see the context here, we are describing visuals actually not talking of visual as a property. 

 `Visuals` are the main building block for UI components and provide reusable rendering logic that is controlled using properties.

I would rather remove code styling in this case, let me know your opinion.

Reply:
@nawab-samsung 

As like the case of https://github.com/Samsung/tizen-docs/blob/55fac53d038491960d7b6e931cabf13bb5c68906/docs/application/dotnet/guides/nui/flexcontainer.md, 
(This link is just an example)
usually each guide page explains one API module (and sub items to that module), and the module is listed in the API.

Indeed, we strongly recommend to set API / Guide set, which means if an API is created, a guide for that API is requested. Thus in the ACR, we ask to create new guide page for a module, if the module is newly created. 

In case of `Visual`, as it is marked with code tag, which implies the function name or code, I searched it from the NUI API, https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.html. As you can also search there is "Visual" under NUI API page, which linked to https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Visual.html.

If you want to use the "Visual" as just the notion of a kind of UI component, 
I agree that not using code tag is proper usage in this sentence.

Reply:
@mijin85cho thanks!